package.json

```diff
@@ -81,7 +81,7 @@
     "lodash-compat": "^3.10.0",
     "nomnoml": "^0.3.0",
     "rollup": "^2.36.1",
-    "rollup-plugin-worker-factory": "0.5.5",
+    "rollup-plugin-worker-factory": "0.5.7",
```

Comment:
Had to update `rollup-plugin-worker-factory` to call `onmessage` in `dispatchEvent` as we use `onmessage` in this scenario in tests.